docs/index.md

```diff
@@ -1,14 +1,58 @@
-# Lockbox Extension
+# Lockbox for desktop
 
-*This is just one component of the Lockbox product. Please see the
-[Lockbox website][org-website] for more documentation and context.*
+<div class="right">
 
-Click below to install the Lockbox extension:
+  <a href="https://testpilot.firefox.com/files/lockbox@mozilla.com/latest"
```

Comment:
Should this all be HTML or Markdown?

Reply:
>Should this all be HTML or Markdown?

I changed it to all HTML to quickly hack in the styles.

I liked having the extension as a .md because GitHub.com would actually render the html still (versus .html which it just renders as formatted source). Not a biggie to me either way, I guess. 🤷‍♂️